ethvm/main.cpp

```diff
@@ -226,7 +246,7 @@ int main(int argc, char** argv)
 	};
 
 	executive.initialize(t);
-	if (contractCodeString != "")
+	if (!contractCodeString.empty())
```

Comment:
wait. by  "will create a contract with init code A" you mean it will execute transaction that will create contract by code A?  

OR 

actual creation of a contract by using setCode  directly in the state?